src/psyclone/tests/psyir/nodes/intrinsic_call_test.py

```diff
@@ -149,6 +150,25 @@ def test_intrinsiccall_datatype(fortran_reader):
     call = psyir.walk(IntrinsicCall)[0]
     assert isinstance(call.datatype, UnresolvedType)
 
+    # ValueError test.
```

Comment:
Please extend the comment to say that you deliberately don't use the `create` method so as to make a malformed IntrinsicCall.